### PR TITLE
Add SRResize

### PR DESCRIPTION
**Pull request by Yshuo-Li**

### CODE REVIEW

File: mmedit/datasets/pipelines/re_sampling.py [hunk 1]
```diff
@@ -7,6 +7,73 @@
 from ..registry import PIPELINES
 
 
+@PIPELINES.register_module()
+class ReSampling:
+    """Resize image.
```
Comment (innerlee) [inline mmedit/datasets/pipelines/re_sampling.py:12]: What's the difference between resample and resize

Comment (Yshuo-Li) [reply mmedit/datasets/pipelines/re_sampling.py:12]: No difference here.

File: mmedit/datasets/pipelines/resize.py [hunk 2]
```diff
@@ -81,9 +151,11 @@ def __call__(self, results):
 
     def __repr__(self):
         repr_str = self.__class__.__name__
-        repr_str += (f'scale_min={self.scale_min}, '
+        repr_str += (f' scale_min={self.scale_min}, '
```
Comment (innerlee) [inline mmedit/datasets/pipelines/resize.py:154]: Remove the extra space

File: mmedit/datasets/pipelines/sr_resize.py [hunk 3]
```diff
@@ -7,6 +7,76 @@
 from ..registry import PIPELINES
 
 
+@PIPELINES.register_module()
+class SRResize:
+    """Resize image, include up sampling and down sampling.
```
Comment (innerlee) [inline mmedit/datasets/pipelines/sr_resize.py:12]: upsampling and  downsampling

File: mmedit/datasets/pipelines/sr_resize.py [hunk 4]
```diff
@@ -7,6 +7,76 @@
...
+
+        The result will be saved in the same key (output_key == input_key)
```
Comment (innerlee) [inline mmedit/datasets/pipelines/sr_resize.py:14]: Image will be loaded from the input_key and the result will be saved in the specified output_key (can equal to input_key).

File: mmedit/datasets/pipelines/sr_resize.py [hunk 5]
```diff
@@ -81,9 +151,11 @@ def __call__(self, results):
 
     def __repr__(self):
         repr_str = self.__class__.__name__
-        repr_str += (f'scale_min={self.scale_min}, '
+        repr_str += (f' scale_min={self.scale_min}, '
```
Comment (innerlee) [inline mmedit/datasets/pipelines/sr_resize.py:154]: Remove extra space

Comment (innerlee) [reply mmedit/datasets/pipelines/sr_resize.py:154]: I see, you can use the f-str format

```python
repr_str = (f'{self.__class__.__name__} ({scale_min={self.scale_min}, '
                  f'scale_max={self.scale_max}, '
                  ...)
```

Comment (Yshuo-Li) [reply mmedit/datasets/pipelines/sr_resize.py:154]: It is consistent with the commonly used description in the repo and can be revised in the future.

**Review by innerlee — APPROVED** [review]

**Comment by innerlee** [comment]
Conflicts with master now

**Pull request merged**